app/views/post_history/post.html.erb

```diff
@@ -1,8 +1,12 @@
+<% @show_content = !!defined?(show_content) ? show_content : true %>
```

Comment:
not sure what `!!defined?` does (double negation?) but I assume that's my failing -- just mentioning for other reviewers to double-check this line

Reply:
This is a safe check for a variable being provided at all (when calling `render` with the template that uses the variable) and assigning a default otherwise, we use the same pattern in many partials. `defined?` prevents a runtime error.